poc/tests/test_idpf_bbcggi21.py

```diff
@@ -84,46 +87,160 @@ def run_idpf_exhaustive_test(self, idpf: Idpf, alpha: int) -> None:
     def test(self) -> None:
         self.run_idpf_test(
             IdpfBBCGGI21(2, 16),
-            0b1111000011110000,
+            (
+                True, True, True, True, False, False, False, False,
+                True, True, True, True, False, False, False, False,
+            ),
```

Comment:
Suggestion: `0b1111000011110000` is a nicer visual representation than the tuple of bools. For testing purposes, it might be nice to add a method for decoding a `tuple[bool, ...]` from an `int`

```suggestion
            prefix_from_int(0b1111000011110000),
```

Reply:
Bonus: it would make it easier to make sure these tests haven't changed.